### PR TITLE
feat(draw): add count parameter

### PR DESCRIPTION
**Pull request by philpax**
This implements a `count` parameter that will make the bot generate multiple images, like "Batch count" does in the web UI.

### CODE REVIEW

File: core/settingscog.py [hunk 1]
```diff
@@ -73,6 +81,10 @@ async def settings_handler(self, ctx,
                 settings.update(guild, 'default_steps', set_maxsteps)
                 reply = reply + '\nDefault steps value is too high! Lowering to ' + str(set_maxsteps) + '.'
 
+        if set_count != 1:
```
Comment (Pyrotuna) [inline core/settingscog.py:84]: This line here implies that it can't be set back to 1 once changed

Comment (Kilvoctu) [reply core/settingscog.py:84]: That appears to be the case. I changed count to 2 and it won't go back to 1.
The settings summary becomes confused too
image

**Comment by Kilvoctu** [comment]
I did consider adding this feature before because the API makes it pretty easy now. I use the batch count feature in the Web UI often, too.
However, I wasn't sure if I liked the idea that one person in a server can possibly sit in queue with like 100 images and everyone else has to wait on that. 🤔 The quick and easy thing to do, I suppose is to add `max_count` in the settings. I'll have to think a bit on how this can be implemented.

**Comment by philpax** [comment]
Fair enough, let me fix both of those issues...

**Comment by philpax** [comment]
Fixed and brought up to the latest `main` 🙂

**Comment by Kilvoctu** [comment]
Very nice, I think everything looks good to me. I appreciate the work. 👍 I like extra flair of the output saying "my # drawings"
There's a very minor bug with the ❌ reaction on batches; I'll fix that and update the readme. Thanks again 👍👍👍

The /settings command is proving to be quite useful

**Pull request merged**